main.go

```diff
@@ -27,11 +28,7 @@ import (
 )
 
 var (
-	description string     = "draughtsman is an in-cluster agent that handles Helm based deployments."
-	f           *flag.Flag = flag.New()
-	gitCommit   string     = "n/a"
-	name        string     = "draughtsman"
-	source      string     = "https://github.com/giantswarm/draughtsman"
+	f *flag.Flag = flag.New()
```

Comment:
These and related changes are due to breaking changes in `microkit` and `microendpoint` made back in July (see Slack [thread](https://gigantic.slack.com/archives/C04TGHDEF/p1563961136007200)).

Reply:
Ack. Good catch and draughtsman doesn't get updated very often.